ruby_on_rails/sidekiq.md

```diff
@@ -76,7 +76,7 @@ bundle install
 schedule_file = 'config/schedule.yml'
 
 if File.exist?(schedule_file) && Sidekiq.server?
-  errors = Sidekiq::Cron::Job.load_from_hash(YAML.load_file(schedule_file))
+  errors = Sidekiq::Cron::Job.load_from_hash! YAML.load_file(schedule_file)
```

Comment:
🐛 

* https://www.rubydoc.info/gems/sidekiq-cron/Sidekiq%2FCron%2FJob.load_from_hash
* https://www.rubydoc.info/gems/sidekiq-cron/Sidekiq%2FCron%2FJob.load_from_hash!

The one with the bang-operator removes the ones, that are not in the list (you can try it out with `Sidekiq::Cron::Job.all.map(&:name)`), while the current one just added the new ones.

Reply:
I'd encourage you to leave the parentheses stay.

Reply:
Agree.